f/flutter-survey-metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"_comments": [
```

Comment:
Nit: I haven't found any of our `.json` files that use tabs (they use spaces). Is there a need to use tabs for `f/*.json` files?

Reply:
Interesting. I googled tabs and JSON, and it seems like it can be problematic. But I have no vote in this race.  :)

Reply:
I did not know that. Let me fix it!

Reply:
Done! I just removed the tabs just in case.